src/Server/Tasks/Numeral.fs

```diff
@@ -5,12 +5,30 @@ open FSharp.Control.Tasks.V2
 open Giraffe
 open Microsoft.AspNetCore.Http
 
-open Core.Numerals
 open Server.Tasks.Utils
- 
+open Core.Numerals
+open Common.Utils
+open Common.Numerals
+
+let getMinMaxRange = function
```

Comment:
I have recently started adding `private` access modifiers to code, you can do too :)